backend/src/main/java/com/rikishi/rikishi/model/Fight.java

```diff
@@ -0,0 +1,45 @@
+package com.rikishi.rikishi.model;
+
+import com.rikishi.rikishi.model.entity.Duel;
+
+public record Fight(
+    long id,
+    User firstUser,
+    User secondUser,
+    int number,
+    WeightClass weightClass,
+    // -1 if no winner. Yes I know it's an anti-pattern, don't have time to correct it
+    long winnerId
+) implements Indexable<Long> {
+    public static Fight fromJson(
+        Duel duel,
+        User resolvedFirstUser,
```

Comment:
I am not sure, what does exactly resolvedFirstUser and resolvedSecondUser does store. Is this just two contestants in the fight?

Reply:
Yeah, just that